crates/bevy_winit/src/accessibility.rs

```diff
@@ -0,0 +1,195 @@
+use std::{
+    collections::VecDeque,
+    num::NonZeroU128,
+    sync::{Arc, Mutex},
+};
+
+use accesskit::{ActionHandler, ActionRequest, Node, NodeId, Role, TreeUpdate};
+use accesskit_winit::Adapter;
+use bevy_app::{App, CoreStage, Plugin};
+use bevy_derive::{Deref, DerefMut};
+use bevy_ecs::{
+    prelude::{Component, Entity, EventReader, EventWriter},
+    query::{Changed, With},
+    system::{NonSend, NonSendMut, Query, RemovedComponents, Res, ResMut, Resource},
+};
+use bevy_utils::{default, HashMap};
+use bevy_window::{WindowClosed, WindowFocused, WindowId};
+
+#[derive(Component, Clone, Default, Deref, DerefMut)]
+pub struct AccessibilityNode(pub Node);
+
+impl From<Node> for AccessibilityNode {
+    fn from(node: Node) -> Self {
+        Self(node)
+    }
+}
+
+#[derive(Default, Deref, DerefMut)]
+pub struct Adapters(pub HashMap<WindowId, Adapter>);
+
+impl Adapters {
+    pub fn get_primary_adapter(&self) -> Option<&Adapter> {
+        self.get(&WindowId::primary())
+    }
+}
+
+#[derive(Resource, Default, Deref, DerefMut)]
+pub struct Handlers(pub HashMap<WindowId, WinitActionHandler>);
+
+#[derive(Clone, Default, Deref, DerefMut)]
+pub struct WinitActionHandler(pub Arc<Mutex<VecDeque<ActionRequest>>>);
+
+impl ActionHandler for WinitActionHandler {
+    fn do_action(&self, request: ActionRequest) {
+        println!("Pushing {:?}", request);
```

Comment:
Please remove any spurious `println`s and unnecessary logs.

Reply:
Necessary for now, unfortunately. That's in a WIP commit and I'm trying to track down why NVDA no longer lets me route my mouse pointer to widgets in the UI demo. It'll be removed before this exits draft.